src/main/java/com/maschel/roomba/RoombaJSSC.java

```diff
@@ -448,7 +448,7 @@ public void motorsPWM(int mainBrushPWM, int sideBrushPWM, int vacuumPWM) throws
      *                       Intermediate values are intermediate intensities.
      * @throws IllegalArgumentException One of the arguments is out of bounds.
      */
-    public void leds(boolean debris, boolean spot, boolean dock, boolean check_robot, int powerColor,
+    public void relativeLeds(boolean debris, boolean spot, boolean dock, boolean check_robot, int powerColor,
```

Comment:
Althoug we could be discussing the necessity of this method:
```
// This would be an easy way to use the existing method in your code
int nonRelativeColor = 165;
int nonRelativeIntensity = 50;
int relativePowerColor = nonRelativeColor/255 * 100;
int relativePowerIntensity = nonRelativeIntensity/255 * 100;
leds(true, true, true, true, relativePowerColor, relativePowerIntensity);
```
If you do want to add it I'm fine with that, but please: **Update and run the Unit Tests!**
I will not merge your pull request without proper unit tests.

If you change the 'leds' method to 'relativeLeds' you have to **edit the 'assertLedsArgumentValidation' and 'testLedsByteOutput' test methods to use the new name of the method**.

Also please add new Unit tests for the added leds method, you can use the 'testLedsArgumentValidation' and 'testLedsByteOutput' as an example.